definitions/divorce/json/CaseEvent/CaseEvent.json

```diff
@@ -2943,5 +2943,18 @@
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/received-service-added-date",
     "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "28/07/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "makeServiceDecision",
+    "Name": "Make Service Decision",
+    "Description": "Make Service Decision",
+    "DisplayOrder": 12,
+    "PreConditionState(s)": "awaitingServicePayment",
```

Comment:
If you have problems not having the states only in "non-prod", that's because this event also needs to only exist in "non-prod"